Wordle.Tests/WordleUnitTest.cs

```diff
@@ -26,13 +30,20 @@ public void EvaluateGuess_WhenNoLetterIsInWord_ShouldReturnAllNotInWord()
 
         var actualResult = game.EvaluateGuess(answer, incorrectGuess);
 
-        Assert.Equal(expectedResult, actualResult);
+        foreach (LetterScore ls in actualResult)
+        {
+            Assert.Equal(expectedResult[ls.Id], ls.Eval);
+        }
     }

```

Comment:
There is a pattern here repeated in these tests, specifically around the `expectedResult` and this `foreach` comparison.  Could you extract a method here to create a new method? Something like `AssertEqualScore(expectedResult, actualResult)`?

Reply:
I like this idea! Apparently xUnit let's you create "Bespoke" assertions so I'll look into this.